src/components/headerAlerts/HeaderAlerts.tsx

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+
+interface HeaderAlertsProps {
+  pathname: string;
+}
+
+export const HeaderAlerts = ({ pathname }: HeaderAlertsProps): JSX.Element | null => {
+  switch (pathname) {
+    case '/explore/api/appeals-status/docs':
+      return (
+        <va-alert background-only show-icon status="info" visible>
+          <p className="vads-u-margin-y--0">
+            A new version of Appeals Status API (v1) will launch later this year.
+          </p>
+        </va-alert>
+      );
+    case '/explore/api/va-facilities/docs':
+      return (
+        <va-alert background-only show-icon status="info" visible>
+          <p className="vads-u-margin-y--0">
+            Version 1 of the VA Facilities API is launching soon. We will add{' '}
+            <Link to="/release-notes/facilities">release notes</Link> when it&apos;s live.
```

Comment:
Need to update to the new release notes page url

```suggestion
            <Link to="/explore/api/va-facilities/release-notes">release notes</Link> when it&apos;s live.
```